xigt/scripts/xigt_export.py

```diff
@@ -4,7 +4,7 @@
 import logging
 from xigt.codecs import xigtxml
 
-def run(infile, outpath, out_format, config=None):
+def run(infile, outpath, out_format, relations, config=None):
```

Comment:
It would be better to have `relations=None` so the argument is optional and to place it after `config=None` so as to not break backward compatibility.